XRTK.SDK/Packages/com.xrtk.sdk/Runtime/Features/UX/Scripts/Controllers/Hands/HandControllerJointsVisualizer.cs

```diff
@@ -11,11 +11,15 @@ namespace XRTK.SDK.UX.Controllers.Hands
     /// <summary>
     /// Hand controller visualizer visualizing hand joints.
     /// </summary>
-    [System.Runtime.InteropServices.Guid("0B653537-D237-4622-ACC6-E351209A5882")]
+    [System.Runtime.InteropServices.Guid("9335E51F-52AB-48EB-A4DC-25832B6B56D6")]
```

Comment:
Why did we change this?

Reply:
I for sure didn't change this myself. Not sure what happened here. But we can just revert the GUID.